### ISSUE TITLE
Update my personal information page

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#13891]
### User story
As a Veteran, I want to understand how I can update my information from the Personal Information page. 

### Acceptance Criteria

1. The "How do I update my personal information?" hyperlink routes to a new page 
2. Styling and UI matches the design.
3. Each contains a hyperlink to "Find your nearest VA medical center" or "Find your nearest Vet Center" (routing to these pages is not needed for this story)

### Designs
Launch page: [https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/431636868](url)
How do I update my personal information? page: [https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/431636872]

### Questions
Routing for the 2 hyperlinks?

**Comment by bessgreen** [comment]
Questions: 

1. Routing for the 2 hyperlinks in AC #3?

**Comment by bessgreen** [comment]
Hi @bridgethapner, for AC # 4-5, can you confirm that **VA Health** and **Vet Centers** are the correct pre-filters?

**Comment by bessgreen** [comment]
Hi @bridgethapner:

1. Are these pages supposed to open in-app, or were these candidates to open out-of-app in a browser?
2. If those drop-downs need to be pre-selected when the pages open, we'll need to loop in the VA.gov team who owns Facility Locator to see if we can pass a query parameter to make these selections.

**Comment by bridgethapner** [comment]
Hi @bessgreen -- 

1. Yes, for AC 4 and 5 those are the correct pre-filters. 
2. I'm working to define all the in and out of app views in ticket #14200 but haven't gotten to it yet...but I think these will be candidates for out of app browser if we go the pre-selected route.
3. Good to know re the drop-downs -- I have them as preselected because that's how VA.gov handles it, however for MVP in our app we can push to Facility Locator generally and include text explaining how to filter. I'm leaning towards that so as to make this a quick decision without looping in another team -- seems the faster route. Maybe we can discuss with @srmyers17 in the product sync today.

**Comment by srmyers17** [comment]
I agree with not pre-selecting the filters for MVP but I will create a ticket for it and put it in the backlog.

**Comment by digitalken** [comment]
Testing:

1. Verify the "How do I update my personal information?" hyperlink routes to a new page.
2. Verify bottom nav bar is removed from this page.
3. Verify styling and UI matches design.
4. Verify there is a working hyperlink to "Find your nearest VA medical center" or "Find your nearest Vet Center."
  a. Verify hyperlink opens in the browser, out of app to https://www.va.gov/find-locations
5. Verify everything on the page is read by the screen reader.
6. Verify each page supports the largest font on the device without loss of content, content overlap or horizontal scroll.
7. Verify all touch point targets are a minimum of 44px by 44px.
8. Verify the page supports operability by tap or voice control only.
9. Verify user can traverse the page(s) by an auxiliary keyboard reader and focus on elements is clear.
10. Verify when a page opens, the first section or element should automatically be placed in focus.
11. Verify that when a page opens, the title page should be announced.

**Comment by digitalken** [comment]
Added to TestRail.

**Comment by esther-k** [comment]
UI and functionality pass. Accessibility does not pass with a multi-issue bug entered. Moving this to Design QA for review

[Accessibility Bug](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/16221)
